packages/common/package.json

```diff
@@ -15,7 +15,7 @@
     "@opentelemetry/api": "^1.4.1",
     "@temporalio/proto": "file:../proto",
     "long": "^5.2.0",
-    "ms": "^2.1.3",
+    "ms": "^3.0.0-canary.1",
```

Comment:
Canary?

Reply:
Yeah... That's not an error.

'ms' has been on "3.0.0-canary.1" since Sept 2021, with >50K weekly downloads. Maintainer is saying they're waiting for a bit more adoption before marking it as stable...